src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/features/ProtocolFunctions.kt

```diff
@@ -21,47 +21,36 @@
 
 package net.ccbluex.liquidbounce.integration.interop.protocol.rest.v1.features
 
-import com.google.gson.JsonArray
-import com.google.gson.JsonObject
 import io.netty.handler.codec.http.FullHttpResponse
-import net.ccbluex.liquidbounce.config.gson.util.emptyJsonObject
+import net.ccbluex.liquidbounce.config.gson.interopGson
 import net.ccbluex.liquidbounce.utils.client.defaultProtocolVersion
 import net.ccbluex.liquidbounce.utils.client.protocolVersion
 import net.ccbluex.liquidbounce.utils.client.protocolVersions
 import net.ccbluex.liquidbounce.utils.client.selectProtocolVersion
 import net.ccbluex.netty.http.model.RequestObject
+import net.ccbluex.netty.http.util.httpNoContent
 import net.ccbluex.netty.http.util.httpOk
 
 // GET /api/v1/protocols
 @Suppress("UNUSED_PARAMETER")
-fun getProtocols(requestObject: RequestObject) = httpOk(JsonArray().apply {
-    for (protocol in protocolVersions) {
-        add(JsonObject().apply {
-            addProperty("name", protocol.name)
-            addProperty("version", protocol.version)
-        })
-    }
-})
+fun getProtocols(requestObject: RequestObject) = httpOk(interopGson.toJsonTree(protocolVersions))
 
 // GET /api/v1/protocols/protocol
 @Suppress("UNUSED_PARAMETER")
-fun getProtocol(requestObject: RequestObject) = httpOk(JsonObject().apply {
-    addProperty("name", protocolVersion.name)
-    addProperty("version", protocolVersion.version)
-})
+fun getProtocol(requestObject: RequestObject) = httpOk(interopGson.toJsonTree(protocolVersion))
```

Comment:
This might lead to someone breaking many scripts without even noticing.

When someone refactors `ClientProtocolVersion`, they might change the names of the fields. They would not expect that this also changes the JavaScript API's behaviour. Please revert this change